index.html

```diff
@@ -392,8 +388,9 @@ <h2>
         following steps MUST be performed:
       </p>
       <ol>
-        <li data-tests="wakelock-cancel-twice.https.html">If the <a>cancel()</a> method has already been invoked on this
-        object, abort these steps.
+        <li data-tests="wakelock-cancel-twice.https.html">If the
+        <a>cancel()</a> method has already been invoked on this object, abort
```

Comment:
We probably want a proper state machine here (i.e., internal slot to track this) - and we probably want to throw an `InvalidStateError` if called twice.